pkg/skaffold/schema/latest/config.go

```diff
@@ -721,18 +721,12 @@ type DockerDeploy struct {
 // KubectlDeploy *beta* uses a client side `kubectl apply` to deploy manifests.
```

Comment:
nit: I believe that from the PR `deploy.Kustomize.Paths` should also be removed but this only touches the `deploy.Kubectl` configuration